recipe/meta.yaml

```diff
@@ -30,17 +30,20 @@ requirements:
     - setuptools
 
   run:
+    - beautifulsoup4
+    - chardet
+    - conda  >=4.1,<4.3
```

Comment:
Should be fine for now. I'll keep you posted on when conda build gets fixed for this.